lib/pf/Switch/Ruckus/Ruckus_Legacy.pm

```diff
@@ -0,0 +1,80 @@
+package pf::Switch::Ruckus::Ruckus_Legacy;
```

Comment:
I'll use similar excuse when people will be reviewing my stuff now

Like:
Oh I was going to write the code properly but then @extrafu and @lzammit transformed the zammitcorp office into a warzone. I really didn't meant to plug in a reboot in the middle of the pfdns boot process